src/components/data-exploration/heatmap/HeatmapExpressionValuesSettings.jsx

```diff
@@ -1,52 +0,0 @@
-import React from 'react';
-import { useSelector, useDispatch } from 'react-redux';
-import { Radio } from 'antd';
-import PropTypes from 'prop-types';
-
-import { updatePlotConfig } from '../../../redux/actions/componentConfig';
-
-const HeatmapExpressionValuesSettings = (props) => {
```

Comment:
why do we no longer need the ExpressionValueSettings?

Reply:
Because we only show one type of expression values, as agreed on with vicky and alex

Reply:
cool, that should already help with the speed of the heatmap too